README.md

```diff
@@ -156,7 +134,7 @@ The following examples uploads local Uri resource, configured to run immediately
 For security reasons, mobile applications cannot contain the full account credentials, and so they cannot freely upload resources to the cloud.
 Cloudinary provides two different mechanisms to enable end-users to upload resources without providing full credentials.
 
-##### 1. Unsigned uploads using [Upload Presets.](http://cloudinary.com/blog/centralized_control_for_image_upload_image_size_format_thumbnail_generation_tagging_and_more) 
+##### 1. Unsigned uploads using [Upload Presets.](http://cloudinary.com/blog/centralized_control_for_image_upload_image_size_format_thumbnail_generation_tagging_and_more)
```

Comment:
This is linking to a 2014 blog post that probably has tons of no longer accurate info. In general, linking to blog posts is risky as the info in them is never updated.

Suggest linking here instead: android_image_and_video_upload#unsigned_upload

And if any info is missing there, feel free to open a DOC JIRA to improve that section.

Reply:
I've changed to the following link:
https://cloudinary.com/documentation/android_image_and_video_upload